hi-ml/src/health_ml/utils/logging.py

```diff
@@ -333,6 +317,107 @@ def _print(self, message: str) -> None:
             sys.stdout.flush()
 
 
+class MLFlowLogger(LightningLoggerBase):
```

Comment:
PL has a logger alrady: https://pytorch-lightning.readthedocs.io/en/stable/extensions/generated/pytorch_lightning.loggers.MLFlowLogger.html

Reply:
It does but it doesn't set the experiment or start/ retrieve a run for us, and it has a different signature to what we want

Reply:
Can't we then at least use the bits that match (inheritance) and modify?

Reply:
... saying all this without having had a very close look at the differences.

Reply:
It doesn't have any that I can see

Reply:
I can look into this more in a future PR